packages/core/src/tools/generateSites.ts

```diff
@@ -40,6 +41,14 @@ export const generateSites = ({
     let sites: SiteInfo[] = [];
     const sitesFilePath = path.resolve(destinationPath ?? DEFAULT_SITES_DIST_PATH);
 
+    debug.multisite('Starting site generation process');
```

Comment:
A couple points I want to add here:
- `site generation process` can be confusing. You can use `site information generation`, `fetching and saving site info` etc
- IMO there's too much unneeded info here. Users can find out whether context id and the rest of API configuration are present in sitecore.config already
- It might also be better to follow a similar approach we use in middleware. If multisite is disabled - log that operation is being skipped. If enabled - inform that operation started execution. Could be more informative this way

Reply:
@MenKNas let's not log context id and other stuff, but simply indicate whether site info generation was activated or not (and why - i.e. multisite is disabled in config)

Reply:
Also there's no need to add both debug and regular logs, I think